CHANGES.md

```diff
@@ -30,6 +30,13 @@ Unreleased
 - Switch back to threaded console for all systems; fix unresponsive console on
   Windows (#7906, @nojb)
 
+3.8.2 (unreleased)
```

Comment:
Add this to the unreleased changelog, @emillon will clean up the changelog later.

Reply:
Ok, but that seems more work @emillon no? As things are now you have to do nothing, right? If I put this in `unreleased` you need to amend both the backport and `main` which seems a huge pain.

Reply:
no, please keep it to the unreleased changelog with no mention of 3.8.2, it's simpler like that. I'll expand on that later.

Reply:
Changelog updated!